packages/fast-components-react-base/src/listbox-item/listbox-item.tsx

```diff
@@ -24,25 +25,60 @@ class ListboxItem extends Foundation<
         disabled: false,
     };
 
+    /**
+     * Gets the id to use for the item
+     * Authors who's items all have unique values can choose to
+     * not provide an itemId and the values of the items will be used as id's,
+     * otherwise use itemId.  All listbox items of a particular listbox should use the same
+     * approach with respect to using values or itemId or results could be unexpected.
+     * The "id" prop is being depracated and should not be used.
+     */
+    public static getItemId = (props: ListboxItemProps): string => {
+        return !isNil(props.itemId)
+            ? props.itemId
+            : !isNil(props.id)
+                ? props.id
+                : props.value;
+    };
+
+    /**
+     * Gets the id to use for an listbox item by examining the associated Element
+     */
+    public static getItemIdFromElement = (element: Element): string => {
+        let idToUse: string = element.getAttribute("itemId");
```

Comment:
This shouldn't actually work, https://developer.mozilla.org/en-US/docs/Web/API/Element/getAttribute

The `itemId` is enumerated in handled props and `getAttribute` only works on attributes that are placed on the element. In that case `itemId`, even if it weren't removed by including it in `handledProps` shouldn't get placed on the element since it is not valid HTML.

Can you add tests for this?